atoma-service/src/server.rs

```diff
@@ -208,140 +205,24 @@ pub async fn run_server(
 /// ```rust,ignore
 /// app.route("/health", get(health_check))
 /// ```
+
+pub const HEALTH_PATH: &str = "/health";
+#[derive(OpenApi)]
+#[openapi(paths(health))]
+pub(crate) struct HealthOpenApi;
+
 #[utoipa::path(
     get,
-    path = "/health",
```

Comment:
why ?

Reply:
Because the path is already being defined here https://github.com/atoma-network/atoma-node/blob/98d20ddefb9308b4b6694696c98dbbf1f1ab26e6/atoma-service/src/server.rs#L211